test/functional/hackable_url_test.rb

```diff
@@ -2,7 +2,13 @@
 
 class HackableUrlTest < ActiveSupport::TestCase
```

Comment:
Turns out the test still passes even if I don't specifically exclude the removed index page. I'll open a separate PR to remove this test.